apps/cli/package.json

```
@@ -27,11 +27,13 @@
     "package:win": "pkg . --targets win-x64 --output ./dist/windows/bw.exe",
     "package:mac": "pkg . --targets macos-x64 --output ./dist/macos/bw",
     "package:lin": "pkg . --targets linux-x64 --output ./dist/linux/bw",
+    "package:lin:arm64": "pkg . --targets linux-arm64 --output ./dist/linux-arm64/bw",
     "debug": "node --inspect ./build/bw.js",
     "dist": "npm run build:prod && npm run clean && npm run package",
     "dist:win": "npm run build:prod && npm run clean && npm run package:win",
     "dist:mac": "npm run build:prod && npm run clean && npm run package:mac",
     "dist:lin": "npm run build:prod && npm run clean && npm run package:lin",
+    "dist:lin:arm64": "npm run build:prod && npm run clean && npm run package:lin:arm64",
```

Here are the changes to the `package.json`:

```suggestion
    "package:lin:arm64": "pkg . --targets linux-arm64 --output ./dist/linux-arm64/bw",
    "debug": "node --inspect ./build/bw.js",
    "dist": "npm run build:prod && npm run clean && npm run package",
    "dist:win": "npm run build:prod && npm run clean && npm run package:win",
    "dist:mac": "npm run build:prod && npm run clean && npm run package:mac",
    "dist:mac:arm64": "npm run build:prod && npm run clean && npm run package:mac:arm64",
    "package:lin:arm64": "pkg . --targets linux-arm64 --output ./dist/linux-arm64/bw",
    "dist:lin": "npm run build:prod && npm run clean && npm run package:lin",
    "dist:lin:arm64": "npm run build:prod && npm run clean && npm run package:lin:arm64",
```